pkgs/servers/jellyseerr/default.nix

```diff
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, mkYarnPackage
+, nodejs
+, nodePackages
+, sqlite
+, fetchYarnDeps
+, python3
+}:
+
+let
+  pin = lib.importJSON ./pin.json;
+in
+
+mkYarnPackage rec {
+  pname = "jellyseerr";
+  version = pin.version;
+
+  src = fetchFromGitHub {
+    owner = "Fallenbagel";
+    repo = "jellyseerr";
+    rev = "v${version}";
+    sha256 = pin.srcSha256;
+  };
+
+  packageJSON = ./package.json;
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = "${src}/yarn.lock";
+    sha256 = pin.yarnSha256;
+  };
+
+  doDist = false;
+
+  nativeBuildInputs = [
+    nodejs
+    makeWrapper
+    sqlite
+    python3
+  ];
+
+  # Fixes "SQLite package has not been found installed" at launch
+  pkgConfig.sqlite3.postInstall = ''
+    export CPPFLAGS="-I${nodejs}/include/node"
+    ${nodePackages.node-pre-gyp}/bin/node-pre-gyp install --prefer-offline --build-from-source --python=${python3}/bin/python --nodedir=${nodejs}/include/node
+  '';
+
+  pkgConfig.bcrypt.postInstall = ''
+    export CPPFLAGS="-I${nodejs}/include/node"
+    ${nodePackages.node-pre-gyp}/bin/node-pre-gyp install --prefer-offline --build-from-source --python=${python3}/bin/python --nodedir=${nodejs}/include/node
```

Comment:
This isn't cross friendly, right?

Reply:
This is the fix for the error in https://github.com/NixOS/nixpkgs/pull/200599. Is there a way to make it cross-friendlier?